### PR TITLE
Update install scripts

### PR DESCRIPTION
**Pull request by mesmith75**
Update to a newer python version with LCG100. Also add in numpy and pandas for users.

### CODE REVIEW

File: ganga/GangaRelease/tools/ganga-cvmfs-install.sh [hunk 1]
```diff
@@ -2,18 +2,22 @@
...
 
 . $1/bin/activate
 
 pip install --upgrade pip setuptools
 
 pip install git+https://github.com/ganga-devs/ganga.git@$1#egg=ganga[LHCb] 
 
+pip install numpy
+
+pip install pandas
```
Comment (alexanderrichards) [inline ganga/GangaRelease/tools/ganga-cvmfs-install.sh]: do we depend on / need numpy or pandas?

**Review by alexanderrichards — DISMISSED** [review]

File: ganga/GangaRelease/tools/ganga-cvmfs-install.sh (hunk 1 above)
Comment (mesmith75) [reply ganga/GangaRelease/tools/ganga-cvmfs-install.sh]: No - but some users have requested and it may be useful for them

Comment (alexanderrichards) [reply ganga/GangaRelease/tools/ganga-cvmfs-install.sh]: ok

Comment (mesmith75) [reply ganga/GangaRelease/tools/ganga-cvmfs-install.sh]: Actually I don't like this - which should add the LCG version to the `PYTHONPATH` instead.

**Review by egede — DISMISSED** [review]
Given that his is the cvmfs installation that is affected by this, I guess it is OK to add pandas and numpy. However, it is a bit of a slippery slope.

**Comment by mesmith75** [comment]
I think this is what we want. For some useful packages you need the compiled libraries so I have put in a little hack to add them to the `LD_LIBRARY_PATH` and then start ganga in a new process. The advantage is it won't change the environment for the user when they quit ganga.

**Review by egede — DISMISSED** [review]

**Review by alexanderrichards — APPROVED** [review]

**Pull request merged**